sdk/resourcemanager/azure-resourcemanager-appservice/src/test/java/com/azure/resourcemanager/appservice/FunctionAppsTests.java

```diff
@@ -104,9 +102,12 @@ public void canCRUDFunctionApp() throws Exception {
                 functionAppResource1.appSettings.get(KEY_AZURE_WEB_JOBS_STORAGE).value(),
                 functionAppResource1.appSettings.get(KEY_CONTENT_AZURE_FILE_CONNECTION_STRING).value());
         // verify accountKey
-        Assertions
-            .assertEquals(
-                functionAppResource1.storageAccount.getKeys().get(0).value(), functionAppResource1.accountKey);
+        String accountKey = functionAppResource1.storageAccount.getKeys().get(0).value();
+        if (!"REDACTED".equals(accountKey)) {
```

Comment:
if (!isPlaybackMode()) {

Reply:
This judgment is fixed to `if (!isPlaybackMode())`